GitUpKit/Core/GCFunctions.m

```diff
@@ -131,6 +135,25 @@ BOOL GCGitOIDFromSHA1Prefix(NSString* prefix, git_oid* oid, NSError** error) {
   return url.absoluteString;
 }
 
+int GCExchangeFileData(const char* path1, const char* path2) {
```

Comment:
This doesn’t seem to be the right to name for a function that can rename the file at `path1` to `path2`. Original name at least suggested that the file is supposed to be temporary.